packages/react/src/components/Storage/FileUploader/__tests__/FileUploader.test.tsx

```diff
@@ -39,4 +47,121 @@ describe('File Uploader', () => {
 
     expect(svg).toBeInTheDocument();
   });
+
+  it('Previewer is not shown on empty file target', async () => {
+    const { container, queryByText } = render(
+      <FileUploader
+        level="public"
+        acceptedFileTypes={['.png']}
+        variation="button"
+      />
+    );
+
+    const input = container.getElementsByTagName('input')[0];
+    await waitFor(() => {
+      fireEvent.change(input, {
+        target: { files: [] },
+      });
+    });
```

Comment:
I don't think you need to `waitFor` for fireEvent, because it fires instaneously.

Reply:
Deleted, it seems to pass without it.

Reply:
Updated in fc15c01